agent/config/runtime.go

```diff
@@ -739,6 +739,12 @@ type RuntimeConfig struct {
 	// time, the connection will be closed by the server.
 	GRPCKeepaliveTimeout time.Duration
 
+	// EnableXDSLoadBalancing controls whether xDS load balancing between servers
```

Comment:
```suggestion
	//  EnableXDSLoadBalancing controls xDS load balancing between the servers.
```
Since the name of the config is EnableXDSLoadBalancing, it should say it controls load balancing.

Reply:
Addressed